src/mysql-util/src/schemas.rs

```diff
@@ -359,7 +359,7 @@ fn parse_data_type(
                     intended_type: None,
                 })?,
         }),
-        "text" | "mediumtext" | "longtext" => Ok(ScalarType::String),
+        "text" | "tinytext" | "mediumtext" | "longtext" => Ok(ScalarType::String),
```

Comment:
I also added in the `tinytext` value here which I previously forgot to include. Will be tested in https://github.com/MaterializeInc/materialize/pull/25852